sources/packages/web/src/components/students/applicationTracker/Completed.vue

```diff
@@ -1,35 +1,92 @@
 <template>
+  <!-- Student appeal - waiting approval -->
+  <application-status-tracker-banner
+    v-if="assessmentDetails.appealStatus === StudentAppealStatus.Pending"
+    label="Your requested change is in progress"
+    icon="fa:fas fa-exclamation-triangle"
+    icon-color="warning"
+    background-color="warning-bg"
+    content="StudentAid BC is reviewing your requested change. If your requested change is approved, your application will be re-evaluated with a new assessment below."
+  />
+  <!-- Student appeal - declined -->
+  <application-status-tracker-banner
+    v-if="assessmentDetails.appealStatus === StudentAppealStatus.Declined"
+    label="Your requested change was declined"
+    icon="fa:fas fa-exclamation-circle"
+    icon-color="danger"
+    background-color="error-bg"
+    content="StudentAid BC has determined an outcome with 1 or more of your requested changes. You can review the outcomes of your requested changes in the table below by clicking “View request”. Please note your application will proceed without your requested changes, based on your last assessment."
+  />
+  <!-- Student appeal - approved -->
+  <application-status-tracker-banner
+    v-if="
+      assessmentDetails.assessmentTriggerType ===
+      AssessmentTriggerType.StudentAppeal
+    "
+    label="Your requested change was approved! Please review your new assessment."
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
+    content="StudentAid BC has determined an outcome with 1 or more of your requested change. Please review your new assessment in the table below."
+  />
+  <!-- Scholastic standing changed -->
+  <application-status-tracker-banner
+    v-if="
+      assessmentDetails.assessmentTriggerType ===
+      AssessmentTriggerType.ScholasticStandingChange
+    "
+    label="You have a new assessment due to your scholastic standing"
+    icon="fa:fas fa-check-circle"
+    icon-color="success"
+    :background-color="hasDisbursementEvent ? undefined : 'success-bg'"
+    content="Your institution informed us of your scholastic standing, which changed your assessment evaluation. Please review your new assessment in the table below. You can also click “View request” to see the details from your institution. Please contact the Financial Aid Officer from your institution, if you have questions about your scholastic standing."
+  />
+  <!-- Offering changed -->
+  <application-status-tracker-banner
+    v-if="
+      assessmentDetails?.assessmentTriggerType ===
```

Comment:
why null check here for assessmentDetails?

Reply:
I followed the others in place (`multiple-disbursement-banner`, `disbursement-banner`) but it can be removed 😉

Reply:
Changed.